lib/org/chromium/webidl/ast/Named.js

```diff
@@ -14,4 +14,10 @@ foam.CLASS({
       name: 'name',
     },
   ],
+
+  methods: [
+    function getName() {
+      return this.name.literal;
```

Comment:
If we want to be more defensive here, we could:

(1) Return `this.name ? this.name.literal || '' : ''`
(2) Implement `validate()` on `Definition` with a call to `this.SUPER()` and a `foam.assert()`ion that `id` is truthy
(3) Call `.validate()` before sending ASTs to a box or putting them to a DAO